source/ada/lsp-lal_utils.adb

```diff
@@ -764,4 +773,74 @@ package body LSP.Lal_Utils is
       return List;
    end List_Bodies_Of;
 
+   ------------------------------
+   -- Check_If_Is_Enum_Literal --
+   ------------------------------
+
+   function Check_If_Is_Enum_Literal
+     (Node      : Ada_Node'Class;
+      Trace     : GNATCOLL.Traces.Trace_Handle;
+      Imprecise : out Boolean) return Boolean
+   is
+      Definition : Defining_Name;
+   begin
+      if Ada_Node (Node) /= No_Ada_Node
+        and then Node.Kind in Ada_Name
+      then
+         Definition := Resolve_Name (Node.As_Name, Trace, Imprecise);
+         if Definition /= No_Defining_Name
+           and then Definition.P_Basic_Decl.Kind = Ada_Enum_Literal_Decl
+         then
```

Comment:
I think this if block could be replaced with

     return (Definition /= No_Defining_Name
           and then Definition.P_Basic_Decl.Kind = Ada_Enum_Literal_Decl);

right?